docs/subnets/create-a-vm-blobvm.md

```diff
@@ -0,0 +1,1672 @@
+# Create a Blob Virtual Machine (VM)
```

Comment:
High-level: 

1. I think we're going into too much detail in some parts of this doc. We should make a clear distinction between what must be implemented as part of a VM (e.g the VM's public interface), vs what was optionally added into the BlobVM and why.
2. We should run this through a grammar checker to get rid of awkward phrasing/run-on sentences/etc. I've used Grammarly and it's pretty nice. There might be some VS-Code plugin for it (I don't use VS code but I think the majority of the team does. Saw [znck](https://marketplace.visualstudio.com/items?itemName=znck.grammarly)  and [LTeX](https://marketplace.visualstudio.com/items?itemName=valentjn.vscode-ltex) looked like good candidates on google). 
3. Should we avoid duplicating common stuff in our [vm doc](https://docs.avax.network/subnets/create-a-virtual-machine-vm#introduction) and only put blob-vm specific stuff here? We can probably safely assume the reader has read that doc as a prereq, or post a link at the top of this doc to let the reader know they should be familiar with that doc before reading this.

Reply:
Initially, we were planning to replace the timestamp tutorial, that's why you were seeing duplications.